src/initiatives/process.md

```diff
@@ -0,0 +1,57 @@
+# Process
+
+This page describes how lang team initiatives work. This is the process to use if you have an idea for a change you would like to make in the language.
+
+## Summary
+
+In a nutshell, the process for a successful initiative is as follows:
+
+- Have an idea
+  - Talk about it on internals, Zulip, etc to flesh it out a bit
+  - Ideally, identify a potential [owner]
+- Open a [proposal] as an issue on the lang-team repository
+  - A lang team member can decide to be your [liaison] and [_second_ your proposal][2nd].
+  - Once that happens, we will create a Zulip stream, tracking issue, and (optionally) repository, etc.
+- If warranted, [explore][experimental] the design space and author the RFC
+  - In this phase, the [owner] works with the [liaison] and other contributors to expore the design space and develop the RFC
+  - Code can be landed in this phase, but the feature gate is marked as "experimental"
+  - Users of the feature gate will get a warning that the RFC is under development
+  - Once the RFC is ready, it can be opened on the RFC repository and approved by the lang team
+- Finish [development]
+  - At this point, development proceeds but the feature gate does not have to be marked as "experimental"
+- [Feature complete]
+  - When the liaison feels that it is ready, the initiative may be declared "feature complete".
+  - This is primarily a 'signaling' mechanism to encourage testing and feedback.
+  - This is a good phase in which to write the Rust reference chapter and other supporting documentation.
+  - Presuming feedback is positive, a stabilization report is prepared and (hopefully) approved.
+- [Stabilized]
+  - Done! The Zulip stream and group can stick around as a place to report bugs or for further discussion, but the initiative is complete.
+  - The final step is to conduct a retrospective discussion between the [owner] and [liaison] about how the process went.
```

Comment:
FWIW I think it likely makes sense to have an official "sunset" phase after stabilization, e.g., asking for the retrospective and such to finalize in ~2 months and then to archive the streams and other repositories/material associated with the group. That way we don't end up with a scattering of groups. I think non-active groups are not generally a good idea to keep around passively. For example, there's an implicit desire to keep the number of groups a liaison is "managing" low -- and this would work counter to that, if groups stick around indefinitely.

Converting to a ping group or similar low-footprint entity may make sense for some initiatives.

Reply:
Hmm. I agree with indicating somehow that the team is inactive and folks are "freed from obligation". Keeping the zulip stream seems potentially useful, but at the same time, if nobody is monitoring, rather confusing. Not sure how to square these things. 

Converting to a ping group is an interesting idea.  I was thinking that it would be useful if the team repo had a better way to record state changes; the simplest would be "inactive_since = 2021-07-09" or something like that, though it might be nice to have a "history" instead. This would allow inactive groups to be hidden from the webpage or in a separate section or something.

Reply:
cc @joshtriplett who I know has thoughts on this point